x/nexus/types/types.go

```diff
@@ -13,8 +13,9 @@ import (
 const maxBitmapSize = 1 << 15 // 32,768
```

Comment:
Maybe we should shrink the bitmaps too since we don't have any plans to substantially increase the windows?

Reply:
maybe, but not this PR.